### PR TITLE
 Required for ssh in ocp

### PR DESCRIPTION
**Pull request by cooktheryan**
Signed-off-by: Ryan Cook <rcook@redhat.com>

**Describe what this PR does**
Part of resolution for OCP to allow for SSH

**Is there anything that requires special attention?**
No just a once over.

**Related issues:**
Part of OCP issue but does not fully resolve

### CODE REVIEW

**Comment by cooktheryan** [comment]
output of pod created 
```
      securityContext:
        capabilities:
          add:
          - AUDIT_WRITE
          - SYS_CHROOT
```

File: controllers/replicationdestination_controller.go [hunk 1]
```diff
@@ -297,7 +297,10 @@ func (r *rsyncDestReconciler) ensureJob(l logr.Logger) (bool, error) {
 		runAsUser := int64(0)
 		r.job.Spec.Template.Spec.Containers[0].SecurityContext = &corev1.SecurityContext{
 			Capabilities: &corev1.Capabilities{
-				Add: []corev1.Capability{"AUDIT_WRITE"},
+                                Add: []corev1.Capability{
+                                        "AUDIT_WRITE",
+                                        "SYS_CHROOT",
```
Comment (JohnStrunk) [inline controllers/replicationdestination_controller.go:302]: This addition will also be needed for the replication source to handle cases where we have the destination pull from the source (meaning the source would be the one running sshd). That direction doesn't work yet, but... eventually.

**Comment by cooktheryan** [comment]
@JohnStrunk ready for review

**Review by JohnStrunk — APPROVED** [review]
LGTM. Thanks!
Related to #16

**Pull request merged**